htdocs/wp-content/install.php

```diff
@@ -107,216 +119,434 @@ function wp_install( $blog_title, $user_name, $user_email, $public, $deprecated
+
+    if ( is_multisite() ) {
+        // Flush rules to pick up the new page.
+        $wp_rewrite->init();
+        $wp_rewrite->flush_rules();
+
+        $user = new WP_User($user_id);
+        $wpdb->update( $wpdb->options, array('option_value' => $user->user_email), array('option_name' => 'admin_email') );
+
+        // Remove all perms except for the login user.
+        $wpdb->query( $wpdb->prepare("DELETE FROM $wpdb->usermeta WHERE user_id != %d AND meta_key = %s", $user_id, $table_prefix.'user_level') );
+        $wpdb->query( $wpdb->prepare("DELETE FROM $wpdb->usermeta WHERE user_id != %d AND meta_key = %s", $user_id, $table_prefix.'capabilities') );
+
+        // Delete any caps that snuck into the previously active blog. (Hardcoded to blog 1 for now.) TODO: Get previous_blog_id.
+        if ( !is_super_admin( $user_id ) && $user_id != 1 )
+            $wpdb->delete( $wpdb->usermeta, array( 'user_id' => $user_id , 'meta_key' => $wpdb->base_prefix.'1_capabilities' ) );
+    }
+
+    /**
+     * Seravo specific option changes that are typically not part of core WordPress wp_install_defaults function
+     */
+
+    /** @see wp-admin/options-general.php */
+    // Remove standard blog description
+    update_option( 'blogdescription', '' );
+    // Set timezone, date and time format
+    update_option( 'timezone_string', seravo_timezone_string() );
+    update_option( 'date_format', seravo_date_format() );
+    update_option( 'time_format', seravo_time_format() );
+
+    /** @see wp-admin/options-reading.php */
+    // Set the created page to show on the homepage
+    update_option( 'show_on_front', 'page' );
+    update_option( 'page_on_front', 1 );
+
+    /** @see wp-admin/options-discussion.php */
+    //Allow people to post comments on new articles (this setting may be overridden for individual articles): false
+    update_option( 'default_comment_status', 0 );
+
+    /** @see wp-admin/options-permalink.php */
+    // Permalink custom structure: /%postname%
+    update_option( 'permalink_structure', '/%postname%/' );
+    
+    /** Activate some plugins automatically if they exists */
+    seravo_install_activate_plugins();
+}
+
+/**
+ * Helper functions to return language specific content and options
+ */
+
+function seravo_page_title() {
+
+    $env_wp_env = getenv('WP_ENV');
+    $env_wp_lang = getenv('WP_LANG');
+
+    if ( 'production' != $env_wp_env ) {
+        $seravo_page_title = 'Welcome';
+    } elseif ( 'fi' == get_locale() || 'fi' == $env_wp_lang ) {
+        $seravo_page_title = 'Tervetuloa';
+    } elseif ( 'sv_SE' == get_locale() || 'sv_SE' == $env_wp_lang ) {
+        $seravo_page_title = 'Välkommen';
+    } else {
+        $seravo_page_title = 'Welcome';
+    }
+
+    return $seravo_page_title;
+}
+
+function seravo_first_page() {
+
+    $env_wp_env = getenv('WP_ENV');
+    $env_wp_lang = getenv('WP_LANG');
+
+
+    if ( 'production' != $env_wp_env ) {
+
+        ob_start();
+        ?>
```

Comment:
Can we have line wrapping here please? It will be easier later on to work with the text and version control if there is not just one very long line.

Reply:
I've got soft wrap set up just the way I like in my editor so this hasn't been a huge issue to me so far but checking the code on GitHub is admittedly a bit painful this way. Do we have any recommendations for how long the lines should be?